src/components/constants.ts

```diff
@@ -1,2 +1,12 @@
 export const PLAYER_MINIMUM_GAMES_REQUIRED = 10;
-export const COMMANDER_MINIMUM_GAMES_REQUIRED = 5;
\ No newline at end of file
+export const COMMANDER_MINIMUM_GAMES_REQUIRED = 5;
+
+export const MTG_COLOR_NAMES = ["Black", "Green", "Red", "Blue", "White"];

```

Comment:
suggestion, we'll probably want a mapping of single letter to full names at somepoint. not required atm.